app/facades/games_facade.rb

```diff
@@ -0,0 +1,31 @@
+class GamesFacade
```

Comment:
This all looks like it makes sense. It will probably break my hardcoded questions, categories, answers when we merge but this looks correct to me. Will need to see when back end is completed.